Changes to README.md to improve readability. (#9899)
mautic/mautic: README.md

[![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors-)
<!-- ALL-CONTRIBUTORS-BADGE:END -->

[![codecov](https://codecov.io/gh/mautic/mautic/branch/features/graph/badge.svg)](https://codecov.io/gh/mautic/mautic)
<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
[![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors-)
<!-- ALL-CONTRIBUTORS-BADGE:END -->

About Mautic
============

> Mautic is the world’s largest open-source marketing automation project. With over 200,000 organisations using Mautic and over 1,000 community volunteers, we empower businesses by making it easy to manage their marketing across a range of channels. Stay up to date about initiatives, releases and strategy via our [blog](https://www.mautic.org/blog).

Reviewer: Suggest we are consistent with using open source as non-hyphenated.

Also we are trying to refactor our links to use macros as we go ([here](https://docs.mautic.org/en/contributing/style-guide#links) is some more info in the style guide).

Will add the macro at the end of the doc!

```suggestion
Mautic is the world’s largest open source marketing automation project. With over 200,000 organisations using Mautic and over 1,000 community volunteers, we empower businesses by making it easy to manage their marketing across a range of channels. Stay up to date about initiatives, releases and strategy via our [blog][mautic-blog].
```

Author: Agreed.

> Marketing automation has historically been difficult to implement within organisations. The Mautic community is an example of open source at its best, offering great software and a vibrant and caring community in which to learn and share knowledge.

Reviewer: Let's capitalise when we are referring to the community as a 'thing'.

```suggestion
Marketing automation has historically been difficult to implement within organisations. The Mautic Community is an example of open source at its best, offering great software and a vibrant and caring community in which to learn and share knowledge.
```

Open source means more than open code. Open source provides equality for all and a chance for everyone to improve.

![Mautic](.github/readme_image.png "Mautic Open Source Marketing Automation")

Get Involved
=============

> Before we tell you how to install and use Mautic, we like to shamelessly plug our awesome user and developer communities! Users, start [here](https://www.mautic.org/community/get-involved) for inspiration, or follow us on Twitter [@MauticCommunity](https://twitter.com/MauticCommunity) or Facebook [@MauticCommunity](https://www.facebook.com/MauticCommunity/). And once you’re familiar with using the software, maybe you will share your wisdom with others in our [Slack](https://www.mautic.org/community/get-involved/communication-channels) channel.

Reviewer: Refactoring to use link macros which I have added at the end of the doc!

```suggestion
Before we tell you how to install and use Mautic, we like to shamelessly plug our awesome user and developer communities! Users, start [here][get-involved] for inspiration, or follow us on Twitter [@MauticCommunity][twitter] or Facebook [@MauticCommunity][facebook]. Once you’re familiar with using the software, maybe you will share your wisdom with others in our [Slack][slack] channel.
```

> Calling all devs, testers and tech writers! Technical contributions are also welcome. First, read our [general guidelines](https://contribute.mautic.org/contributing-to-mautic) about contributing. If you want to contribute code, read our [CONTRIBUTING.md](https://github.com/mautic/mautic/blob/feature/.github/CONTRIBUTING.md) or [Contributing Code](https://contribute.mautic.org/contributing-to-mautic/developer) docs then check out the issues with the [L1 label](https://github.com/mautic/mautic/issues?q=is%3Aissue+is%3Aopen+label%3AL1) to get stuck in quickly and show us what you’re made of.

Reviewer: Refactoring to use link macros and updating to T1 rather than L1 - goodness this was out of date! 😊 

```suggestion
Calling all devs, testers and tech writers! Technical contributions are also welcome. First, read our [general guidelines][contributing] about contributing. If you want to contribute code, read our [CONTRIBUTING.md][contributing-md] or [Contributing Code][contribute-developer] docs then check out the issues with the [T1 label][t1-isssues] to get stuck in quickly and show us what you’re made of.
```

> If you have questions, the Mautic community can help provide the answers.

Reviewer: ```suggestion
If you have questions, the Mautic Community can help provide the answers.
```

> Installing and Using Mautic

Reviewer: Let's not use camel case in headings

```suggestion
Installing and using Mautic
```

|3.3   | 16 Feb 2021 | 22 Feb 2021 | 17 May 2021 | 24 May 2021
|4.x   | 17 May 2021 | 24 May 2021 | 24 May 2022 | 20 Dec 2022

`*`Security support for 2.16 will only be provided for Mautic itself, not for core dependencies that are EOL, such as Symfony 2.8.

## Software Downloads

> The GitHub version is recommended for both development and testing. The production package (including all libraries) is available at [mautic.org/download](https://www.mautic.org/download).

Reviewer: Refactor to use link macros

```suggestion
The GitHub version is recommended for both development and testing. The production package (including all libraries) is available at [mautic.org/download][download-mautic].
```

## Installation
### Disclaimer

> *Install from source only if you are comfortable using the command line. You'll be required to use various CLI commands to get Mautic working and keep it working. If the source/database schema gets out of sync with Mautic releases, the release updater may not work and will require manual updates. For production, we recommend the pre-packaged Mautic which is available at [mautic.org/download](https://www.mautic.org/download).*

Reviewer: Refactor to use existing link macro that was used in my edit above (reduces duplication!)

```suggestion
*Install from source only if you are comfortable using the command line. You'll be required to use various CLI commands to get Mautic working and keep it working. If the source/database schema gets out of sync with Mautic releases, the release updater may not work and will require manual updates. For production, we recommend the pre-packaged Mautic which is available at [mautic.org/download][download-mautic].*
```

> *Also note that source code outside of a [tagged release](https://github.com/mautic/mautic/releases) should be considered ‘alpha’. It may contain bugs, cause unexpected results, data corruption or loss, and is not recommended for use in a production environment. Use at your own risk.*

Reviewer: Refactor for link macro

```suggestion
*Also note that source code outside of a [tagged release][tagged-release] should be considered ‘alpha’. It may contain bugs, cause unexpected results, data corruption or loss, and is not recommended for use in a production environment. Use at your own risk.*
```

> ### How to Install Mautic

Reviewer: Let's not use camel case in headings

```suggestion
### How to install Mautic
```

> You must already have [Composer v1](http://getcomposer.org/) available on your computer because this is a development release and you'll need Composer to download the vendor packages. Note that Composer v2 is not yet supported.

Reviewer: ```suggestion
You must already have [Composer v1][composer-v1] available on your computer because this is a development release and you'll need Composer to download the vendor packages. Note that Composer v2 is not yet supported.

Also note that if you have DDEV installed, you can run 'ddev config' followed by 'ddev start'. This will kick off the Mautic first-run process which will automatically install dependencies and configure Mautic for use. ✨ 🚀 Read more [here][ddev-mautic]
```

Author: @RCheesley - suggested revision:

Also note that if you have DDEV installed, you can run 'ddev config' followed by 'ddev start'. This will kick off the Mautic first-run process which will automatically install dependencies and configure Mautic for use. ✨ 🚀 Read more [here][ddev-mautic].

Installing Mautic is a simple three-step process:

> 1. [Download the repository zip](https://github.com/mautic/mautic/archive/master.zip) then extract the zip to your web root.

Reviewer: Added the link macro and also updated the URL - as we now have the features branch as default the old master URL will not work (and another 😱  at how outdated that link is!)

```suggestion
1. [Download the repository zip][download-zip] then extract the zip to your web root.
```

2. Run the `composer install` command to install the required packages.
3. Open your browser and complete the installation through the web installer.

> If you get stuck, check our our [general troubleshooting](https://docs.mautic.org/en/troubleshooting) page. Still no joy? Join the [Mautic community](https://www.mautic.org/community) for support and answers.

Reviewer: ```suggestion
If you get stuck, check our our [general troubleshooting][troubleshooting] page. Still no joy? Join our lively [Mautic Community][community] for support and answers.
```

Author: @RCheesley should be [Mautic Community] (capital C). I assume you want to capitalise 'Community' in both contexts?

Join our lively Community.
Join the Mautic Community.

Reviewer: Great catch!!

> ### User Docs

Reviewer: Prefer if we use the full term rather than abbreviation in headings :) 

```suggestion
### User Documentation
```

Author: Ha - sure. That's what happens when technical writers try to get funky. :)

> Documentation on how to use Mautic is available at [docs.mautic.org](https://docs.mautic.org).

Reviewer: ```suggestion
Documentation on how to use Mautic is available at [docs.mautic.org][mautic-docs].
```

### Developer Docs

> Developer documentation, including API reference docs, is available at [developer.mautic.org] (https://developer.mautic.org/).

Reviewer: ```suggestion
Developer documentation, including API reference docs, is available at [developer.mautic.org][dev-docs].
```

<!-- ALL-CONTRIBUTORS-LIST:END -->

> This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!

Reviewer: Adding the link macros at the end of the page!

```suggestion
This project follows the [all-contributors][all-contributors] specification. Contributions of any kind welcome!

[mautic-blog]: <https://www.mautic.org/blog>
[get-involved]: <https://www.mautic.org/community/get-involved>
[twitter]: <https://twitter.com/MauticCommunity>
[facebook]: <https://www.facebook.com/MauticCommunity/>
[slack]: <https://www.mautic.org/community/get-involved/communication-channels>
[contributing]: <https://contribute.mautic.org/contributing-to-mautic>
[contributing-md]: <https://github.com/mautic/mautic/blob/feature/.github/CONTRIBUTING.md>
[contribute-developer]: <https://contribute.mautic.org/contributing-to-mautic/developer>
[t1-issues]: <https://github.com/mautic/mautic/issues?q=is%3Aissue+is%3Aopen+label%3AT1>
[download-mautic]: <https://www.mautic.org/download>
[tagged-release]: <https://github.com/mautic/mautic/releases>
[composer-v1]: <http://getcomposer.org/>
[download-zip]: <https://github.com/mautic/mautic/archive/refs/heads/features.zip>
[ddev-mautic]: <https://kb.mautic.org/knowledgebase/development/how-to-install-mautic-using-ddev>
[troubleshooting]: <https://docs.mautic.org/en/troubleshooting>
[community]: <https://www.mautic.org/community>
[mautic-docs]: <https://docs.mautic.org>
[dev-docs]: <https://developer.mautic.org>
[all-contributors]: <https://github.com/all-contributors/all-contributors>
```

Author: @RCheesley I didn't edit the final paragraph as the instructions were to not update. But it could use some help.

This project follows the [all-contributors specification](https://github.com/all-contributors/all-contributors). Contributions of any kind are welcome!
